pyevmasm/evmasm.py

```diff
@@ -71,483 +71,283 @@ class InstructionTable(dict):
         Instruction(0x3, 'SUB', 0, 2, 1, 3, 'Subtraction operation.', None, 0)
 
     """
-
-    def __init__(self, *args, **kwargs):
+    __slots__ = ('_previous_fork', '_cache')
+    def __init__(self, *args, previous_fork=None, **kwargs):
```

Comment:
Was getting an error with this in python2. Switching to something like this seems to be ok. What do you think?
```
    def __init__(self, *args,  **kwargs):
        super(InstructionTable, self).__init__(*args, **kwargs)
        self._previous_fork = kwargs.get('previous_fork', None)
        self._cache = {}
```